package.json

```diff
@@ -51,5 +62,6 @@
     "**/diff": "^3.5.0",
     "**/growl": "1.10.2",
     "**/minimist": "^1.2.3"
-  }
+  },
+  "dependencies": {}
```

Comment:
Was this added for a specific reason?

Reply:
No idea how this got there will revert, good catches 🙂

Reply:
Oh looks like this got there from my accidentally adding husky as a `dependency` and not a `devDependency`